parm/config/gfs/config.base

```diff
@@ -176,9 +176,9 @@ export DO_OCN="NO"
 export DO_ICE="NO"
 export DO_AERO="NO"
 export DO_PREP_OBS_AERO="NO"
-export AERO_FCST_RUN="" # When to run aerosol forecast: gdas, gfs, or both
-export AERO_ANL_RUN="" # When to run aerosol analysis: gdas, gfs, or both
-export WAVE_RUN="" # When to include wave suite: gdas, gfs, or both
+export AERO_FCST_RUN="gdas" # When to run aerosol forecast: gdas, gfs, or both
+export AERO_ANL_RUN="both" # When to run aerosol analysis: gdas, gfs, or both
+export WAVE_RUN="both" # When to include wave suite: gdas, gfs, or both
```

Comment:
I like this!  So much simpler than hunting down the correct location within `APP =~ ...A` if blocks.

Reply:
Same!